src/ol-ext/wfsdescribefeaturetype.js

```diff
@@ -103,55 +167,67 @@ ol.format.WFSDescribeFeatureType.NAMESPACE_URIS_ = [
 
 /**
  * @const
- * @type {!Object.<string, !Object.<string, ol.XmlParser>>}
+ * @type {Object.<string, !Object.<string, ol.XmlParser>>}
```

Comment:
Isn't it working with that?
https://github.com/camptocamp/ngeo/commit/1855a97818208432ab1fb25d34a8edeafb138f1e